### DISCUSSION TITLE
Nuclei as replacement for Greenbone?

### DISCUSSION THREAD

**Discussion by jhjacobs81** [projectdiscovery/nuclei-templates#6944] [General]
So i am looking at Nuclei, and so far it looks rather nice. Can it be used as a replacement for Greenbone vulnerability scanning? I think the answer is yes, but can someone confirm this? Before i start learning all the ins and outs :)

**Comment by ehsandeep** [comment]
@jhjacobs81 Nuclei is a powerful tool for automated detection of security issues such as vulnerabilities, misconfigurations, and other potential weaknesses in web applications. While it shares some similarities with vulnerability scanners like Greenbone, it is not a direct replacement for them.

Nuclei can be a useful addition to a comprehensive security testing toolset, and can help to augment the coverage provided by vulnerability scanners like Greenbone. It is important to note, however, that Nuclei is a relatively new tool and may have a different level of maturity and support Greenbone has.

Ultimately, the best approach is to use a combination of tools that provide a comprehensive view of the security posture of your web applications and network infrastructure. You can start by incorporating Nuclei into your existing security testing workflows and evaluating its effectiveness for your specific use case.

Nuclei Docs - https://nuclei.projectdiscovery.io/
Nuclei Foundation (video series) - https://www.youtube.com/watch?v=b5qMyQvL1ZA&list=PLZRbR9aMzTTpItEdeNSulo8bYsvil80Rl
